environments/change_to_relative_pos.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import numpy as np
```

Comment:
please organize the imports as described [here](https://github.com/hill-a/stable-baselines/blob/master/CONTRIBUTING.md)